src/Compilers/Core/Portable/Emit/EditAndContinue/DeletedEventDefinition.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Cci;
+using Microsoft.CodeAnalysis.Emit.EditAndContinue;
+using Microsoft.CodeAnalysis.Symbols;
+
+namespace Microsoft.CodeAnalysis.Emit
```

Comment:
>Emit [](http://example.com/codeflow?start=33&length=4)

Should this be in `Microsoft.CodeAnalysis.Emit.EditAndContinue` along with the other Deleted* symbols?